templates/definition/charger/hardybarth-salia.yaml

```
@@ -16,6 +16,13 @@ requirements:
   evcc: ["sponsorship"]
 params:
   - name: host
+  - name: user
+    required: false
+  - name: password
+    required: false
+    mask: true
```

```suggestion
  - name: user
  - name: password
```

No need to repeat defaults.yaml definitions